.changelog/4501.internal.md

```diff
@@ -0,0 +1 @@
+docs: Migrate to docusaurus
```

Comment:
Please, use the `doc` Change Log fragment type and reword it to something like:
"Migrate Docs to Docusaurus"